src/libsync/propagateupload.cpp

```diff
@@ -707,6 +735,7 @@ QMap<QByteArray, QByteArray> PropagateUploadFileCommon::headers()
 {
     QMap<QByteArray, QByteArray> headers;
     headers[QByteArrayLiteral("Content-Type")] = QByteArrayLiteral("application/octet-stream");
+    Q_ASSERT(_item->_modtime > 0);
```

Comment:
@mgallien Isn't a log also needed for this assert?

Reply:
yes thanks